documents/minutes/2017-12-12 WCA Board Founding Meeting.md

```diff
@@ -1,80 +1,82 @@
-<div class="text-center">
+::::: {.text-center}
 MINUTES OF MEETING OF THE BOARD OF DIRECTORS OF WORLD CUBE ASSOCIATION
 
 A California Nonprofit Public Benefit Corporation
-</div>
+:::::
 
 On December 12, 2017 at 1:00 p.m. PST, the Board of Directors of World Cube Association (the “Board”) met for its initial meeting via Google Hangouts. The following directors were present at the meeting:
 
-<div class="text-center">
+::::: {.text-center}
 Pedro Santos Guimarães <br>
 Olivér Perge <br>
 Ron van Bruchem <br>
 Alberto Pérez de Rada Fiol <br>
 Luis J. Iáñez <br>
 Robert Burton <br>
-</div>
+:::::
 
 On the motion and by unanimous vote, the following persons were elected temporary chairperson and secretary for the first meeting:
 
 - Temporary Chairperson: Ron van Bruchem
 - Temporary Secretary: Robert Burton
 
-<b class="text-center">MINUTE BOOK</b>
+#### **MINUTE BOOK** {.text-center}
```

Comment:
You're changing this to become a heading (MD `####` element) whereas previously it was just "some text" wrapped in a `b` tag.

Why do you also additionally need to make it bold with `**text**`? Shouldn't headings be bold by themselves?

Reply:
And if they are indeed not bold by default (which would be very surprising!) then have you considered making them bold as part of your custom CSS stylesheet?

Reply:
They're bold by default, that's my bad